resources/views/layouts/app.blade.php

```diff
@@ -10,8 +10,14 @@
     <title>{{ config('app.name', 'Laravel') }}</title>
```

Comment:
add this title and use to other pages
`<title>@yield('title')　|　{{ config('app.name', 'Chronostep Community') }}</title>`
usage for other pages
ex
login.blade.php
`@section('title', 'Login')`

to add css file for other pages
```
    @if (trim($__env->yieldContent('css')))
    <link rel="stylesheet" href="{{ url('css') }}/@yield('css')">
    @endif
```

usage for other pages
ex
login.blade.php
`@section('css', 'login.css')`